handel/src/services/apigateway/swagger/swagger-deploy-type.ts

```diff
@@ -253,8 +288,23 @@ export async function deploy(stackName: string, ownServiceContext: ServiceContex
     lambdasToCreate = await uploadDeployableArtifactsToS3(ownServiceContext, lambdasToCreate, serviceName, enrichedSwagger);
     const swaggerS3ArtifactInfo = await uploadSwaggerToS3(ownServiceContext, enrichedSwagger);
     const compiledTemplate = await getCompiledApiGatewayTemplate(stackName, ownServiceContext, ownPreDeployContext, dependenciesDeployContexts, lambdasToCreate, swaggerS3ArtifactInfo, stackTags);
+
+    await require('fs-extra').writeFile('./apigw.yml', compiledTemplate);
```

Comment:
What is the purpose of this line?

Reply:
The purpose of that line is that I was supposed to remove it before committing 🤦‍♂️